overlord/snapstate/check_snap.go

```diff
@@ -263,13 +263,14 @@ func checkCoreName(st *state.State, snapInfo, curInfo *snap.Info, flags Flags) e
 	// transition we will end up with not connected interface
 	// connections in the "core" snap. But the transition will
 	// kick in automatically quickly so an extra flag is overkill.
-	if snapInfo.Name() == "core" && core.Name() == "ubuntu-core" {
+	// TODO parallel-install: use instance name
```

Comment:
don't think we need the TODO, but we do need to block installing multiple instances of core

Reply:
Ok, I'll leave a TODO in `doInstall()` then.